config/default/kustomization.yaml

```diff
@@ -10,7 +10,7 @@ namePrefix: telemetry-
 
 # Labels to add to all resources and selectors.
 labels:
-- includeSelectors: true
```

Comment:
We dont want to have labelselectors included from kustomize right now. We want to have the old label selectors so that the migration is smooth and we can revert this change when we rename the `operator` to `manager` again